<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllMain</class>
 <widget class="QMainWindow" name="AllMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1063</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1063</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AllMain</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#ptn_close
{
	image: url(:/images/close.png);
	border:none;
}

QPushButton#ptn_close:hover
{
	image: url(:/images/close_.png);
	border:none;
}

QPushButton#ptn_min
{
	image: url(:/images/little.png);
	border:none;
}

QPushButton#ptn_min:hover
{
	image: url(:/images/little_.png);
	border:none;
}

QPushButton#ptn_max
{
	image: url(:/images/max.png);
	border:none;
}

QPushButton#ptn_max:hover
{
	image: url(:/images/max_.png);
	border:none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font : 14px;
	color : rgb(255, 255, 255);
}

</string>
   </property>
   <widget class="QWidget" name="widget_top" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1063</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1063</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1063</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color: rgb(22,134,182);
}

QLabel#label_txt
{
	font: 18px '楷体';
	color:  rgb(255,255,255);
}

QPushButton#ptn_close:hover
{
	background-color: rgb(232,17,35);
}

QPushButton#ptn_min:hover
{
	background-color: rgba(255,255,255,127);
}

QLabel#label_title
{
	image: url(:/images/calabash.png);
}</string>
    </property>
    <widget class="QLabel" name="label_txt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Calabash</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_min">
     <property name="geometry">
      <rect>
       <x>943</x>
       <y>0</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_close">
     <property name="geometry">
      <rect>
       <x>1003</x>
       <y>0</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_curtain" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1063</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1063</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1063</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QWidget
{
	background-color: rgba(22,134,182,178);
}
*/
QLable#label_icon
{
	background-color:transparent;
}</string>
    </property>
    <widget class="QLabel" name="label_icon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>69</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/images/point.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_begin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;开始&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_begin_">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;点击下面按键以使用更多Calabash请假功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_tail" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>1063</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1063</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1063</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color: rgba(22,134,182,165);
}
QLabel#label_username
{
	font: color(255,255,255);
}
QPushButton#ptn_home
{
	
	image: url(:/images/home.png);
}</string>
    </property>
    <widget class="QLabel" name="label_head">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>371</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_home">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>15</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ptn_introduce">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>340</width>
      <height>340</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>340</width>
      <height>340</height>
     </size>
    </property>
    <property name="text">
     <string>小组介绍/模块指南</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>538</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>28</number>
     </property>
     <item>
      <widget class="QPushButton" name="ptn_owner">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string>个人信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ptn_teacher">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string>教师</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ptn_student">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string>学生</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>542</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <widget class="QPushButton" name="ptn_setting">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string>风格切换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ptn_signOut">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string>登出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
